src/Websites/Websites/Utilities/WebsitesClient.cs

```diff
@@ -816,11 +817,10 @@ public Site UpdateHostNameSslState(string resourceGroupName, string webAppName,
                     ToUpdate = true,
                     SslState = sslState
                 }},
-                Location = location
+                Location = location,
+                Tags =  webapp.Tags
```

Comment:
@panchagnula we should preserve the null-check here that we had previously (`webapp?.Tags`)

Reply:
i will add that back.